modules/drivers/bigquery-cloud-sdk/deps.edn

```diff
@@ -3,4 +3,4 @@
 
  :deps
  ;; TODO: figure out how to be able to leave off this version string and use the version from the BOM
- {com.google.cloud/google-cloud-bigquery {:mvn/version "1.131.1"}}}
+ {com.google.cloud/google-cloud-bigquery {:mvn/version "2.6.2"}}}
```

Comment:
this is a big jump

Reply:
yeah. I just wanted to see what would happen TBH

Reply:
@jeff303 was battling problems in #19059, where the dependency versions was lined up between BQ and GA, since it would otherwise break both drivers depending on which one was loaded first.
Just want to avoid something like that happening again.

Reply:
reverted